src/main.jsx

```diff
@@ -14,9 +16,11 @@ import './index.css';
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <Providers>
-      <CssBaseline />
-      <App />
-    </Providers>
+    <Provider store={store}>
```

Comment:
It would be possible to place redux provider inside providers component for a more streamlined configuration, but I think this approach is also working well.